src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,27 @@
+package duke;
+
+/**
+ * Deals with interactions with the user
+ *
+ * @author sibinhho99-nus
+ */
+public class Ui {
+    /**
+     * Prints a String and then terminate the line.
+     */
+    public void println(String message) {
+        System.out.println(message);
+    }
+
+    /**
+     * Prints the formatted version of given message to console
+     */
+    public void printMessage(String message) {
```

Comment:
Maybe could add methods that print specific messages instead of passing the whole message to the method ?
eg. have a method named printWelcome() instead of printMessage("Hi I'm duke")